### DISCUSSION TITLE
Move enabled property to spatie/ray?

### DISCUSSION THREAD

**Discussion by patinthehat** [spatie/ray#261] [Ideas]
@freekmurze: 
I feel that it would be worthwhile to move the `enabled` property on the `Ray` class _(and config settings, etc.)_ from `spatie/laravel-ray` to `spatie/ray`.  This will allow developers to toggle whether or not ray is enabled regardless of the package (base, laravel, wordpress, etc.) they're using.

I'd like to submit a PR for this but want to make sure it is in line with your vision for the package and that the PR will be considered.

Thoughts?

References:
- https://github.com/spatie/laravel-ray/blob/master/src/Ray.php#L29
- https://github.com/spatie/laravel-ray/blob/master/src/Ray.php#L289

**Comment by freekmurze** [comment]
If this can be achieved in a non-breaking way, I'd accept a PR with the necessary modifications.

**Reply by patinthehat** [reply]
> If this can be achieved in a non-breaking way, ...

@freekmurze Did you have any specific concerns or thoughts on this?

**Reply by freekmurze** [reply]
Not really, except that we should try not to break anything with this transition. If all tests pass, I'll assume it works (after doing some manual tests as well).

Thanks for you work on this!
